### ISSUE TITLE
The top1-accuracy in your training log is the validation accuracy or testing accuracy?

### ISSUE THREAD

**Issue by smallcube** [XuZhengzhuo/LiVT#1]
Thanks for the Great work!
I was wandering the accuracy reported in your training log is obtained from validation set or testing set?

**Comment by XuZhengzhuo** [comment]
Hi, it depends on the datasets. ImageNet-LT & iNat18 & CIFAR-LT are the validation set and PlacesLT is the test set. The settings keep consistent with the previous work. Thx for your interest.
